mobileapps/serializers.py

```diff
@@ -9,30 +8,16 @@ class Meta:
 
 
 class MobileAppSerializer(serializers.ModelSerializer):
-    provider_key = serializers.CharField(source="provider_key_decrypted", required=False)
-    provider_secret = serializers.CharField(source="provider_secret_decrypted", required=False)
     deployment_mechanism = serializers.ChoiceField(choices=DEPLOYMENT_CHOICES, required=False)
 
     class Meta:
         model = MobileApp
         extra_kwargs = {'updated_by': {'read_only': True}}
 
     def _set_custom_validated_data(self, validated_data):
-        '''
+        """
         Here we are adding decrypted keys and adding current user in 'updated_by'
```

Comment:
How about removing `adding decrypted keys` from here because now we are only adding current user.